src/main/java/springboot/webapp/usersmanager/CustomGeometry/CustomGeometry.java

```diff
@@ -0,0 +1,12 @@
+package springboot.webapp.usersmanager.CustomGeometry;
+
+import org.springframework.expression.ParseException;
+
+public interface CustomGeometry {
+
+    GeometryType getType();
+
+    default double getArea() {
```

Comment:
why do you need this default method? Area can't be calculated in most geometries